api/test/integration/test_rbac_black_logtest_endpoints.tavern.yaml

```diff
@@ -0,0 +1,42 @@
+---
+test_name: PUT /logtest
+
+marks:
+  - rbac_tests
+
+stages:
+
+  - name: Run logtest (deny)
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/logtest"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+      json:
+        event: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+        log_format: "syslog"
+        location: "master->/var/log/syslog"
+    response: &permission_denied
+      status_code: 403
+      json:
+        error: 4000
+
+---
+test_name: DELETE /logtest/sessions
+
+marks:
```

Comment:
I am almost sure we dont use marks in every test but just once for the entire file, confirm please

Reply:
Confirmed, I made a mistake. Sorry for the confusion.